ecommerce-ui/src/RentalCart/RentalCart.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RentalCart from './RentalCart';
+import TestRenderer from 'react-test-renderer';
+
+describe("RentalCart", () => {
+  it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<RentalCart homesCart={homesCart()} cancelYourRent={() => {}} whatRentalTotal={() => { return 0; }} />, div);
```

Comment:
These functions are not defined. Instead, you'll want to pass some fake data via props.